src/test/videopress/local-helpers/utils.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import {
+	initializeEditor,
+	getBlock,
+	fireEvent,
+	openBlockSettings,
+} from 'test/helpers';
+/**
+ * Internal dependencies
+ */
+import { VIDEOPRESS_BLOCK_HTML } from './constants';
+
+export const initializeBlockWithHTML = async (
+	initialHtml = VIDEOPRESS_BLOCK_HTML
+) => {
+	const screen = await initializeEditor( { initialHtml } );
+
+	return { ...screen };
+};
```

Comment:
Not sure if this function is reducing the test boilerplate 🤔 . It would be the same as doing:
```
const screen = await initializeEditor( { initialHtml: VIDEOPRESS_BLOCK_HTML  } );
```
vs
```
const screen = await initializeBlockWithHTML( VIDEOPRESS_BLOCK_HTML );
```
Do we expect to add more functionality in the future?

Reply:
Ah, you're right, thanks for pointing this out. I don't expect to add further functionality here in the future, I believe it ended up this way due to some local explorations and changes. I've gone ahead to remove this as a helper in https://github.com/wordpress-mobile/gutenberg-mobile/pull/5751/commits/5293b8b128b97f0df329446a693c9c8c48ae4dea.